docs/en/04_Changelogs/4.0.0.md

```diff
@@ -1179,15 +1279,34 @@ handle field-level and form-level messages. This has the following properties:
 * `Requirements::delete_combined_files()` and `Requirements::delete_combined_files()` methods have been removed
  as they are obsolete.
 
-### <a name="overview-mailer"></a>Email and Mailer
+### <a name="overview-i18n"></a>i18n API
 
-* `Mailer` converted to an interface
-* `SwfitMailer` added as new default mailer
-* `Email` re-written to be powered by [SwiftMailer](https://github.com/swiftmailer/swiftmailer)
+#### <a name="overview-i18n-api"></a>i18n API Additions / Changes
+
+* Upgrade of i18n to symfony/translation
+* Localisation based on language-only (without any specific locale) is now supported
+* i18nEntityProvider::provideI18nEntities() Now is expected to return only a single array
+  map of key to default values.
+* i18n keys for '.PLURAL_NAME' and '.SINGULAR_NAME' have been changed back to FQN class names
```

Comment:
FQCN is probably what you mean, but I agree it's best to avoid these abreviations where possible